### ISSUE TITLE
400 / 600 mhz no difference in performance

### ISSUE THREAD

**Issue by dimitre** [sipeed/Maixduino#46]
Hello, I've just tested some benchmark at 400Mhz and 600Mhz and the performance outputs the same. I think this options aren't working on Maixduino yet.
https://github.com/PaulStoffregen/CoreMark
Thanks
![Screen Shot 2019-08-07 at 1 03 05 PM](https://user-images.githubusercontent.com/58289/62638597-2b647880-b914-11e9-8770-54118e54a784.png)
![Screen Shot 2019-08-07 at 1 04 19 PM](https://user-images.githubusercontent.com/58289/62638598-2b647880-b914-11e9-9896-e6881d8271ae.png)

**Comment by CombiesGit** [comment]
`void setup() 
{
  sysctl_cpu_set_freq(F_CPU);
}

void loop() 
{

}`


Are there any side effects?
I dont know.

**Comment by romainreignier** [comment]
Indeed, I have looked into the source code and the menu entry does not attempt to change the CPU frequency anywhere.
It only changes the `SystemCoreClock` constant but it is not used anywhere, not even in the `millis()` implementation.
